Merge.py

```diff
@@ -0,0 +1,40 @@
+
+def merge_sort(array):
+    L = len(array)
+    if L == 1:
+        return array
+    else:
+        part1 = merge_sort(array[0:L//2])
+        part2 = merge_sort(array[L//2:])
+        l1 = len(part1)
+        l2 = len(part2)
+        i1 = 0
+        i2 = 0
+        arrayToReturn = [0]*(l1 + l2)

```

Comment:
Btw, install [**black**](https://github.com/ambv/black) formatter and format your code with it. It's quite handy.
```suggestion
        arrayToReturn = [0] * L
```